app/controllers/dashboard/work_histories_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Dashboard
+  class WorkHistoriesController < BaseController
+    def show
+      @work = current_user.works.find(params[:work_id])
+      @work_history = WorkHistoryPresenter.new(@work)
+      @latest_work_version = WorkVersionDecorator.new(@work.latest_version)
```

Comment:
@rschenk I don't know if this matters, but `@work` doesn't seem to be used in the views anywhere. I'm also wondering if the WorkHistoryPresenter might have a `latest` method that produces the `@latest_work_version`. That would cut down the number of instance variables the controller is having to set.

Reply:
Good thinking on all points!